### ISSUE TITLE
Import playlists from Spotify

### ISSUE THREAD

**Issue by shermaza** [MeoMix/StreamusChromeExtension#345]
It would be nice to be able to import playlists from Spotify so that all of my music could be in one place. I imagine it as first logging into Spotify, selecting a playlist, and for each track in the selected playlist, searching for the title and selecting the best match. I believe this is a relevant page for how to access a user's playlists: 

https://developer.spotify.com/web-api/get-playlists-tracks/

Edit: I did not see issue #236! Basically the same thing.

**Comment by Basaaa** [comment]
Seconded

**Comment by JordanBlakey** [comment]
If it auto syncs to Spotify playlist, super bonus points. Personal use case: Spotify blocked on my work network, which is why I use Streamus at work, as a workaround. 

Blocking Spotify and other streaming music sites is common in the corporate world, with the exception of Youtube, which is often not blocked. Easy migration from Spotify to Streamus and you've opened a whole new can of users. 

Personally I might cancel Spotify premium, since my playlists are the main reason I use it at this point.

**Comment by MeoMix** [comment]
Hey,

Closing this in favor of #236. Feel free to continue any discussions there.
